regression/pages/lms/instructor_dashboard.py

```diff
@@ -0,0 +1,37 @@
+"""
+Instructor dashboard page.
+"""
+from edxapp_acceptance.pages.lms.instructor_dashboard import (
+    InstructorDashboardPage
+)
+from regression.pages.lms import BASE_URL
+
+
+class InstructorDashboardPageExtended(InstructorDashboardPage):
+    """
+    This class is an extended class of Instructor Dashboard Page,
+    where we add methods that are different or not used in
+    InstructorDashboardPage
+    """
+    @property
+    def url(self):
```

Comment:
Is this the same as the base class? If so, you can just use super and you don't need to re-declare it here.